src/main/java/net/sacredlabyrinth/phaed/dynmap/simpleclans/DynmapSimpleClans.java

```diff
@@ -1,181 +1,224 @@
+    }
+
+    /**
+     * Sends a debug message to the console, respecting the user decision.
+     *
+     * @param message             message to send
+     * @param respectUserDecision should the message be sent if debug is false?
+     */
+    public static void debug(String message, boolean respectUserDecision) {
+        if (respectUserDecision && !instance.getConfiguration().getBoolean("debug", false)) {
+            return;
+        }
+
+        instance.getLogger().info(message);
+    }
+
+    public static void debug(String message) {
+        debug(message, true);
+    }
+
+    @Override
+    public void onEnable() {
+        instance = this;
+        saveDefaultConfig();
+        if (reload()) {
+            new CommandManager(this);
+            getPluginManager().registerEvents(new DynmapSimpleClansListener(this), this);
+        }
+    }
+
+    public FileConfiguration getConfiguration() {
+        return configuration;
+    }
+
+    public boolean reload() {
+        reloadConfig();
+        configuration = getConfig();
+
+        try {
+            loadDependencies();
+        } catch (IllegalStateException ex) {
+            getLogger().severe(ex.getMessage());
+            return false;
+        }
+
+        saveDefaultImages(configuration);
+        loadTasks(configuration);
+        loadLayers(configuration);
+
+        return true;
+    }
+
+    public void loadLayers(FileConfiguration config) {
+        ConfigurationSection clanHomesSection = Objects.requireNonNull(config.getConfigurationSection("layer.homes"));
+        ConfigurationSection killsSection = Objects.requireNonNull(config.getConfigurationSection("layer.kills"));
+        ConfigurationSection landsSection = Objects.requireNonNull(config.getConfigurationSection("layer.lands"));
+
+        String defaultHomeIcon = clanHomesSection.getString("default-icon", DefaultIcons.CLANHOME.getName());
+
+        IconStorage homesIcons = new IconStorage(this, "/images/clanhome", defaultHomeIcon, markerApi);
+        IconStorage killsIcons = new IconStorage(this, "/images", DefaultIcons.BLOOD.getName(), markerApi);
+
+        try {
+            homesLayer = new HomesLayer(getClanManager(), homesIcons, new LayerConfig(clanHomesSection), markerApi);
+        } catch (IllegalStateException ex) {
+            debug(ex.getMessage());
+        }
+
+        try {
+            killsLayer = new KillsLayer(killsIcons, new LayerConfig(killsSection), markerApi);
+        } catch (IllegalStateException ex) {
+            debug(ex.getMessage());
+        }
+
+        try {
+            // Running on next ticks to safely receive lands coordinates
+            getScheduler().runTask(this, () ->
+                    landsLayer = new LandsLayer(getClanManager(), simpleclans.getProtectionManager(), new LayerConfig(landsSection), markerApi));
+        } catch (IllegalStateException ex) {
+            debug(ex.getMessage());
+        }
```

Comment:
I thought of `return` in the constructor.
But it seems, I can't instantiate the secondary constructor if it has the same values as the primary one.
Also tried with `run { }` in `init`, but no success here 🤔

Reply:
It's fine as it is.